site/content/docs/0.0/migration.md

```diff
@@ -19,12 +40,12 @@ From now on, OUDS Web won't embed Bootstrap elements that are not part of Orange
 
 #### Opacity
 
-- <span class="badge text-bg-success">New</span> Opacity utilities: `.opacity-transparent`, `.opacity-weaker`, `.opacity-weak`, `.opacity-medium`, `.opacity-emphasis` and `.opacity-opaque`.
+- <span class="badge text-bg-success">New</span> Opacity utilities: `.opacity-transparent`, `.opacity-weaker`, `.opacity-weak`, `.opacity-medium`, `.opacity-strong` and `.opacity-opaque`.
 
 ### CSS and Sass variables
 
 - <span class="badge text-bg-success">New</span> `$enable-bootstrap-compatibility` option set to `false` by default. This option allows you to compile the Sass files with a Bootstrap compatibility mode.
-  - For instance, without the Bootstrap compatibility mode, you won't have the opacity Bootstrap utilities such as `opacity-0`, `opacity-1`, `opacity-50`, etc. You will only have the semantic OUDS Web utilities such as `opacity-transparent`, `opacity-weaker`, `opacity-weak`, etc.
+  - For instance, without the Bootstrap compatibility mode, you won't have the opacity Bootstrap utilities using numbers. You will only have the semantic OUDS Web utilities words instead.
```

Comment:
@vprothais Would you be OK if we revert this change? @hannahiss and I kinda prefer the old one with examples, that seems easier to understand on our side.

Reply:
That's good for me too, I also find the previous one more understandable :)